backend/controllers/api.js

```diff
@@ -1,66 +0,0 @@
-const Book = require("./../models/book.model");
```

Comment:
How is it that we can just remove this file? Is it not being used?

Reply:
Nope, not used at all. Only specialized controllers like book, quote etc. are used. You can checked exposed endpoint in the [docs](https://the-one-api.dev/documentation#4).